README.md

```diff
@@ -29,6 +29,7 @@ operators in production.
 
 ## Docs
 
+- [Alerter Services](docs/alerter_services.md)
```

Comment:
I don't follow the naming here - these are just prometheus collectors, no?

Reply:
Maybe. I don't know. It was once called alerter service as abstracted concept in the `aws-operator`. It is true that we use Prometheus collectors as an implementation detail on our side but we do not want to dictate this. The abstracted concept needs a name though and when you know a better one please suggest one.

Reply:
I would just go with Collectors.

Reply:
I don't want this because I do not want to advocate a specific piece of technology. Again, this is about an abstracted concept, which needs a name.

Reply:
Gave my opinion. All good

Reply:
I always appreciate your opinion. Thank you.

Reply:
If you want an agnostic idea, `Metrics Provider` might be one - I think having alert in there gives the wrong idea (that this is towards alerting, not metrics)

Reply:
Metrics provider makes sense to me. I go with this then.

Reply:
https://media.giphy.com/media/26BRwtmWTmy2xsbmg/giphy.gif